clients/spark/core/src/main/scala/io/treeverse/gc/UncommittedGarbageCollector.scala

```diff
@@ -102,6 +114,9 @@ object UncommittedGarbageCollector {
     }
   }
 
+  // By default, objects that are written in the last 6 hours are not collected
+  private val DEFAULT_GC_UNCOMMITTED_SKIP_SECONDS = 6 * 60 * 60
```

Comment:
You can add it also to the LakeFSContext

Reply:
Done.